front/src/applications/stdcm/views/OSRDStdcmResults.tsx

```diff
@@ -1,15 +1,33 @@
-import React, { useMemo, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 
 import { ChevronDown, ChevronUp } from '@osrd-project/ui-icons';
+import { PDFDownloadLink } from '@react-pdf/renderer';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 
-import type { SimulationReport } from 'common/api/osrdEditoastApi';
+import {
+  osrdEditoastApi,
+  type PostStdcmApiResponse,
+  type SimulationReport,
+} from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
 import SpaceTimeChart from 'modules/simulationResult/components/SpaceTimeChart/SpaceTimeChart';
 import { useStoreDataForSpaceTimeChart } from 'modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart';
 import SpeedSpaceChart from 'modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart';
+import type { OsrdStdcmConfState } from 'reducers/osrdconf/types';
 import type { AllowancesSettings } from 'reducers/osrdsimulation/types';
+import { useAppDispatch } from 'store';
 
-const OSRDStcdmResults = () => {
+import SimulationReportSheet from '../components/SimulationReportSheet';
+import formatStdcmConf from '../formatStdcmConf';
+import { generateCodeNumber } from '../utils';
+
+type OSRDStcdmResultsProps = {
+  mapCanvas?: string;
+};
+
+const OSRDStcdmResults: React.FC<OSRDStcdmResultsProps> = ({ mapCanvas }) => {
```

Comment:
Don't use React.FC for the moment :) 
```suggestion
const OSRDStcdmResults = ({ mapCanvas }: OSRDStcdmResultsProps) => {
```

Reply:
Done